dvc/output.py

```diff
@@ -258,6 +280,7 @@ class Output:
     PARAM_PERSIST = "persist"
     PARAM_REMOTE = "remote"
     PARAM_PUSH = "push"
+    PARAM_CLOUD = "cloud"
```

Comment:
`cloud` or `clouds`?

Reply:
I guess `cloud` makes more sense for now.